subsys/bluetooth/mesh/sensor_types.c

```diff
@@ -546,36 +572,38 @@ FORMAT(energy)           = SCALAR_FORMAT(3,
 /*******************************************************************************
  * Lighting formats
  ******************************************************************************/
-FORMAT(chromatic_distance)      = SCALAR_FORMAT_MAX(2,
-						(SIGNED |
-						 HAS_INVALID |
-						 HAS_UNDEFINED),
-						unitless, SCALAR(1e-5, 0),
-						5000);
+FORMAT(chromatic_distance)      = SCALAR_FORMAT_MIN_MAX(2,
+							(SIGNED |
+							HAS_INVALID |
+							HAS_UNDEFINED),
+							unitless, SCALAR(1e-5, 0),
+							-5000, 5000);
 FORMAT(chromaticity_coordinate) = SCALAR_FORMAT(2,
 						UNSIGNED,
 						unitless,
 						SCALAR(1, -16));
-FORMAT(correlated_color_temp)	= SCALAR_FORMAT(2,
-						(UNSIGNED | HAS_UNDEFINED),
-						kelvin,
-						SCALAR(1, 0));
+FORMAT(correlated_color_temp)	= SCALAR_FORMAT_MIN_MAX(2,
+							(UNSIGNED | HAS_UNDEFINED),
+							kelvin,
+							SCALAR(1, 0),
+							800, 65534);
 FORMAT(illuminance)		= SCALAR_FORMAT(3,
 						(UNSIGNED | HAS_UNDEFINED),
 						lux,
 						SCALAR(1e-2, 0));
-FORMAT(luminous_efficacy)	= SCALAR_FORMAT(2,
-						(UNSIGNED | HAS_UNDEFINED),
-						lumen_per_watt,
-						SCALAR(1e-1, 0));
+FORMAT(luminous_efficacy)	= SCALAR_FORMAT_MAX(2,
+						    (UNSIGNED | HAS_UNDEFINED),
+						    lumen_per_watt,
+						    SCALAR(1e-1, 0),
+						    18000);
 FORMAT(luminous_energy)		= SCALAR_FORMAT(3,
 						(UNSIGNED | HAS_UNDEFINED),
 						lumen_hour,
-						SCALAR(1e3, 0));
+						SCALAR(1, 0));
```

Comment:
With 1e3 it won't fit in `int32_t`. Since resolution is 1000, first 3 orders can be omitted. Then the value fits in `int32_t`. A user will need to divide/multiply the value by 10^3 to encode/decode the actual value.

Reply:
This should be reflected in the header and unit declaration, so that users can tell that this is the case. Unit should be Kilolumen (`klmn`). This change also belongs in release notes for 1.7. Same below